query-engine/connectors/mongodb-query-connector/src/value.rs

```diff
@@ -364,7 +364,7 @@ fn read_scalar_value(bson: Bson, meta: &ScalarOutputMeta) -> crate::Result<Prism
         // DateTime
         (TypeIdentifier::DateTime, Bson::DateTime(dt)) => PrismaValue::DateTime(dt.to_chrono().into()),
         (TypeIdentifier::DateTime, Bson::Timestamp(ts)) => {
-            PrismaValue::DateTime(Utc.timestamp(ts.time as i64, 0).into())
+            PrismaValue::DateTime(Utc.timestamp_opt(ts.time as i64, 0).unwrap().into())
```

Comment:
I'm pasting here the implementation of `Utc::timestamp` for the reviewer's convenience:

```rust
    /// Makes a new `DateTime` from the number of non-leap seconds
    /// since January 1, 1970 0:00:00 UTC (aka "UNIX timestamp")
    /// and the number of nanoseconds since the last whole non-leap second.
    ///
    /// Panics on the out-of-range number of seconds and/or invalid nanosecond,
    /// for a non-panicking version see [`timestamp_opt`](#method.timestamp_opt).
    #[deprecated(since = "0.4.23", note = "use `timestamp_opt()` instead")]
    fn timestamp(&self, secs: i64, nsecs: u32) -> DateTime<Self> {
        self.timestamp_opt(secs, nsecs).unwrap()
    }
```